src/engine/renderer/gl_shader.h

```diff
@@ -3496,7 +3496,7 @@ class u_VertexInterpolation :
 {
 public:
 	u_VertexInterpolation( GLShader *shader ) :
-		GLUniform1f( shader, "u_VertexInterpolation" )
+		GLUniform1f( shader, "u_VertexInterpolation", true )
```

Comment:
This seems wrong. This one is set for surfaces that are part of an MD3 model

Reply:
I suppose that uniform can just be removed from the shader since it will likely use other means to do vertex skinning/animation.

Reply:
I've removed this uniform entirely from the material shaders for now.

Reply:
This change can be removed then since it's in principle a per-surface value.

Reply:
Ah, yea, removed.